docs/tools/cicd/integration_test_failures.md

```diff
@@ -0,0 +1,51 @@
+# Integration Test Failures
+
+There are two (2) to receive notifications of integration test failures. 
+One is in [MilMove CircleCI dashboard](https://app.circleci.com/pipelines/github/transcom/mymove) 
+and the other is from the [GitHub pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) page.
+
+## CircleCI
+
+You will see a `Failed` notification on the dashboard page. Click on the `Failed` notification/label to see what step in the CI/CD has failed.
+![Screen Shot 2021-11-09 at 2 38 48 PM](https://user-images.githubusercontent.com/1359520/141003337-a15e2cb5-2036-4752-af00-85b836dd5a44.png)
+
+After you click on the `Failed` notification/label, you will be presented with a list of actions that were taken or are still in progress for the commit.
+Click on the failed indicator to see the details of what is failing.
+![Screen Shot 2021-11-09 at 2 38 56 PM](https://user-images.githubusercontent.com/1359520/141003369-e26facb8-7f45-4794-acfb-a70f107b879f.png)
```

Comment:
These GitHub User Content images have an expiration date that is unknown to us, but we can expect them exist forever. It's an undocumented GitHub feature to use them as a CDN. 

We need these images in the Repository if we're going to reference them in our documentation.